Helm/opensearch/templates/statefulset.yaml

```diff
@@ -270,14 +270,31 @@ spec:
           - name: node.{{ $role }}
             value: "{{ $enabled }}"
           {{- end }}
+{{- if .Values.extraEnvs }}
+{{ toYaml .Values.extraEnvs | indent 10 }}
+{{- end }}
+{{- if .Values.envFrom }}
+        envFrom:
+{{ toYaml .Values.envFrom | indent 10 }}
+{{- end }}
+        volumeMounts:
+          {{- if .Values.persistence.enabled }}
+          - name: "{{ template "opensearch.uname" . }}"
+            mountPath: /usr/share/opensearch/data
+          {{- end }}
+          {{- if .Values.keystore }}
+          - name: keystore
+            mountPath: /usr/share/opensearch/config/opensearch.keystore
+            subPath: opensearch.keystore
+          {{- end }}
```

Comment:
https://github.com/opensearch-project/opensearch-devops/pull/31/files

Already being fixed here. Just to avoid duplication of work. Maybe consider pulling in after #31 gets merged.

Reply:
Good catch. I second @DandyDeveloper